assets/src/design-system/components/dropDown/select/components.js

```diff
@@ -58,7 +58,7 @@ export const SelectButton = styled.button(
     css`
       ${themeHelpers.focusableOutlineCSS(
         theme.colors.interactiveBg.negativeNormal
-      )};
+      )}
```

Comment:
Do we still need this? Probably not important 😅

Reply:
nope